config/routes.rb

```diff
@@ -3,15 +3,15 @@
 
   devise_for :users
   
-  resources :comments
-  resources :follow_requests
-  resources :likes
-  resources :photos
+  resources :comments, except: [:index, :show]
+  resources :follow_requests, except: [:index, :show, :new, :edit]
+  resources :likes, only: [:create, :destroy]
+  resources :photos, except: [:index]
```

Comment:
Great job. If routes are not supposed to be used, the most secure way to protect your app is to remove these routes. Authorizations/Pundit comes after that.